src/plotman/plot_util.py

```diff
@@ -12,11 +12,13 @@ def df_b(d):
 def get_k32_plotsize():
     return 108 * GB
 
-def human_format(num, precision):
+def human_format(num, precision, powerOfTwo=False):
+    divisor = 1024 if powerOfTwo else 1000
+    
     magnitude = 0
-    while abs(num) >= 1000:
+    while abs(num) >= divisor:
         magnitude += 1
-        num /= 1000.0
+        num /= divisor        
     return (('%.' + str(precision) + 'f%s') %
             (num, ['', 'K', 'M', 'G', 'T', 'P'][magnitude]))
```

Comment:
Sorry this got left hanging...  Does this seem ok to you?  Does seem like it should be indicated.

```suggestion
    result = (('%.' + str(precision) + 'f%s') %
            (num, ['', 'K', 'M', 'G', 'T', 'P'][magnitude]))

    if powerOfTwo and magnitude > 0:
	    result += 'i'
    
    return result
```